sdks/java/core/src/test/java/com/google/cloud/dataflow/sdk/transforms/display/DisplayDataMatchers.java

```diff
@@ -16,13 +16,19 @@
 
 package com.google.cloud.dataflow.sdk.transforms.display;
 
+import static org.hamcrest.Matchers.allOf;
```

Comment:
I chatted with @lukecwik on this; since `DisplayDataMatchers` will be packaged in the test jar, it should be ok. This is the case with the others listed above as well.